File: client/src/app/domain/project-status/project-status.ts
import { EvalueeStatus } from './evaluee-status';
import { FeedbackProviderStatus } from './feedback-provider-status';
import { ReviewerStatus } from './reviewer-status';
import { AdminStatus } from './admin-status';

export class ProjectStatus {
    id: number;
    name: string;
    description: string;
    evalueesStatus: EvalueeStatus[];
    feedbackProvidersStatus: FeedbackProviderStatus[];
    reviewersStatus: ReviewerStatus[];
    adminsStatus: AdminStatus[];
    reminder: true;
}

File: projects/src/main/java/com/gire/eval360/projects/domain/dto/ProjectStatus.java
package com.gire.eval360.projects.domain.dto;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
@EqualsAndHashCode(callSuper = false)
public class ProjectStatus {

	private Long id;
	private Long idTemplate;
	private String name;
	private String description;
	private List<EvalueeStatus> evalueesStatus;
	private List<FeedbackProviderStatus> feedbackProvidersStatus;
	private List<ReviewerStatus> reviewersStatus;
	private List<AdminStatus> adminsStatus;
	
}
